util/use-workspace-package-in-examples.js

```diff
@@ -0,0 +1,34 @@
+import { existsSync, readdirSync, readFileSync, statSync, writeFileSync } from 'fs';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const examplesDir = join(__dirname, '../examples');
+
+// Get all directories in examples folder
+const examples = readdirSync(examplesDir)
+  .filter(file => statSync(join(examplesDir, file)).isDirectory());
+
+// Process each example's package.json
+examples.forEach(example => {
```

Comment:
what exactly does this file do?

Reply:
it sets the `@solidjs/start` package to `workspace:*` in all the examples, so they can be used for local testing.